### PR TITLE
add static pages

### PR DESCRIPTION
**Pull request by higs4281**
Ports static WordPress pages into Django templates
## Additions
- static templates for kbyo and pledge pages with content ported from wordpress
## Changes
- updates urls to point to new templates
## Testing

after pulling in code, you should see the ported pages at  
- /paying-for-college2/student-loan-forgiveness-pledge/
- /paying-for-college2/know-before-you-owe-student-debt/

for reference, the original pages:
- http://www.consumerfinance.gov/pledge/ 
- http://www.consumerfinance.gov/students/knowbeforeyouowe/ 
## Review
- @marteki @ascott1 @niqjohnson

### CODE REVIEW

**Comment by marteki** [comment]
The Know Before You Owe student debt page looks great! We lost the breadcrumb at the top of the page, but that's expected since the page isn't in the same page hierarchy anymore. :+1: for that part.

I'm getting an template error when I try to view the student loan forgiveness pledge, and will comment in the diff where it's being triggered.

File: paying_for_college/templates/pledge-static.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{% extends base_template %}
+{% load staticfiles %}
+{% block page_meta %}
+<meta charset="UTF-8" />
+<meta property="og:title" content="Paying for College &gt; Consumer Financial Protection Bureau"/>
+<meta property="og:type" content="government"/>
+<meta property="og:url" content="{{ URL }}"/>
+{% endblock %}
+
+{% block title %}Take the pledge &gt; Consumer Financial Protection Bureau%}
```
Comment (marteki) [inline paying_for_college/templates/pledge-static.html]: Missing an endblock here?

**Comment by higs4281** [comment]
@marteki fixed!

**Comment by niqjohnson** [comment]
:+1:

Both pages are showing up at the expected URLs in both UnityBox and standalone for me.

**Comment by ascott1** [comment]
Both work great for me. :+1:

**Pull request merged**